alicloud/resource_alicloud_nas_filesystem.go

```diff
@@ -0,0 +1,147 @@
+package alicloud
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/nas"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+func resourceAlicloudNASFilesystem() *schema.Resource {
+	return &schema.Resource{
```

Comment:
PLease rename `resourceAlicloudNASFilesystem ` to `resourceAlicloudNasFileSystem `